packages/contracts-bedrock/src/L2/L2ToL2CrossDomainMessenger.sol

```diff
@@ -145,19 +146,22 @@ contract L2ToL2CrossDomainMessenger is ISemver, TransientReentrancyAware {
         if (_target == Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER) revert MessageTargetL2ToL2CrossDomainMessenger();
 
         uint256 nonce = messageNonce();
+        bytes memory context = "";
```

Comment:
I am confused, should there be a `sendMessage(uint,address,bytes,bytes)` in addition where the context can be set?